influxdb3/tests/server/query.rs

```diff
@@ -1119,10 +1127,12 @@ async fn api_v1_query_chunked() {
                           "values": [
                             [4, "a", 0.5],
                             [5, "a", 0.6],
-                          ]
+                          ],
+                          "partial": true
                         }
                       ],
-                      "statement_id": 0
+                      "statement_id": 0,
+                      "partial": true
```

Comment:
Both of these are incorrect, as there are no more records for the measurement or statement.